YuliiaShaposhnikova/HW7/Task3.py

```diff
@@ -0,0 +1,14 @@
+def how_many_letters(some_word):
```

Comment:
good job
but look at more optimize version of code

def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result